xbmc/epg/EpgDatabase.cpp

```diff
@@ -59,6 +59,12 @@ void CEpgDatabase::CreateTables(void)
         "idEpg           integer, "
         "sTitle          varchar(128), "
         "sPlotOutline    text, "
+        "sIMDBNumber     text, "
+        "sCast           text, "
+        "sDirector       text, "
+        "sWriter         text, "
+        "sOriginalTitle  text, "
+        "sYear           text, "
```

Comment:
you also have to bump the EPG database number in the header, and write the code to update the database to add these new fields.